spec/migrations/20170710132935_custom_button_to_classes_spec.rb

```diff
@@ -0,0 +1,61 @@
+require_migration
+
+describe CustomButtonToClasses do
+  let(:custom_button_stub) { migration_stub(:CustomButton) }
+  let(:miq_set_stub) { migration_stub(:MiqSet) }
+
+  shared_examples 'migrate up' do
```

Comment:
Nice to use `shared_example`.  However IMHO, in this case, I'd rather not to use it because 
1. the `let`s are not easy to follow, plus not saving too much code
2. you won't be able to test the 2nd cases that I suggested.